setup/install.sh

```diff
@@ -111,27 +148,31 @@ else
 fi
 
 echo -e "\x1b[1;34m[*] Installing dotnet for C# agents and modules\x1b[0m"
-if [ $OS_NAME == "DEBIAN" ]; then
+if [ "$OS_NAME" == "DEBIAN 1*" ]; then
   wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
does this need to differentiate between `debian/10` and `debian/11`?

Reply:
It does not ?
"DEBIAN 1*" matches both "DEBIAN 10" and "DEBIAN 11", as well as future versions

Reply:
There should be `https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb` in addition to `https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb`